packages/features/settings/ThemeLabel.tsx

```diff
@@ -18,6 +18,7 @@ export default function ThemeLabel(props: ThemeLabelProps) {
         type="radio"
         value={value}
         id={`theme-${variant}`}
+        data-testid={`theme-${variant}`}
```

Comment:
this should be on the label tag, it seems that the click event doesn't work on an invisible element